scalalib/src/mill/scalalib/GenIdeaImpl.scala

```diff
@@ -34,7 +35,8 @@ object GenIdeaImpl {
 
     val jdkInfo = extractCurrentJdk(pwd / ".idea" / "misc.xml").getOrElse(("JDK_1_8", "1.8 (1)"))
 
-    rm! pwd/".idea"
```

Comment:
this change makes the user's life more comfortable by not asking them to reset various pieces of mill-unrelated configuration